server/gql/models/authors/index.js

```diff
@@ -85,15 +85,20 @@ export const customUpdateResolver = async (model, args, context) => {
       country: args.country,
       age: args.age
     };
-    const authorsBooksArgs = { authorsBooks: args.booksId };
```

Comment:
basis [this](https://github.com/sagar-wednesday/node-express-graphql-template/pull/9/commits/2f7666525b4bb435e2373791a6e3d2b993882a19#diff-d6f533ec5377df0ccb8b0888a608ed349a48370ad5a528c36ae6ef2832ec33d0R17) it seems like the args are not compulsory and hence will not always have a value. 

in this case, for the response we'll need to fetch the fields that are requested in the mutation